pyproject.toml

```diff
@@ -16,7 +16,7 @@ authors = [
 readme = 'README.md'
 
 repository = "https://github.com/NHSDigital/covid-19-test-result"
-homepage = ""
+homepage = "https://github.com/NHSDigital/covid-19-test-result"
```

Comment:
Just to get the github actions passing as it doesnt like an empty string for homepage uri